File: packages/services/core-service/test_db.py
import psycopg2
import sys

url = "postgresql://postgres:JXaYfLedIWpwfXXOFuRkhSityLMAfole@crossover.proxy.rlwy.net:40549/railway"
try:
    conn = psycopg2.connect(url)
    print("Conexão bem-sucedida!")
    conn.close()
except Exception as e:
    print(f"Erro: {e}")
    sys.exit(1)

File: packages/services/core-service/test_db.js
const { Client } = require('pg');
const client = new Client({
  connectionString: "postgresql://postgres:JXaYfLedIWpwfXXOFuRkhSityLMAfole@crossover.proxy.rlwy.net:40549/railway",
});
client.connect()
  .then(() => {
    console.log('Conexão bem-sucedida!');
    return client.end();
  })
  .catch(err => {
    console.error('Erro de conexão:', err.message);
    process.exit(1);
  });
